flink-dist/src/main/flink-bin/bin/pyflink2.bat

```diff
@@ -22,4 +22,4 @@ setlocal EnableDelayedExpansion
 SET bin=%~dp0
 SET FLINK_ROOT_DIR=%bin%..
 
-%FLINK_ROOT_DIR%\bin\flink run -v %FLINK_ROOT_DIR%\lib\flink-python*.jar 2 %*
\ No newline at end of file
+"%FLINK_ROOT_DIR%\bin\flink" run -v "%FLINK_ROOT_DIR%"\lib\flink-python*.jar 2 %*
```

Comment:
that would result in all arguments passed being treated as a single argument.

if additional paths are 
* relative, Flink will receive a relative path and use internal tools to make them absolute.
* absolute, the user has to use "" himself.